src/test/java/nextstep/domain/LineTest.java

```diff
@@ -29,36 +29,21 @@ public void drawTransverseBar(int coutOfPerson, TransverseBarStrategy transverse
         );
     }
 
-    @DisplayName("라인 밸리데이션 - 최소 하나의 횡단선을 가진다.")
-    @ParameterizedTest
-    @MethodSource("validateHasOneOrMoreTransverseBarMethodSource")
-    public void validateHasOneOrMore(int coutOfPerson, TransverseBarStrategy transverseBarStrategy, boolean result) {
-        List<Boolean> line = Line.drawTransverseBar(coutOfPerson, transverseBarStrategy);
-        assertThat(Line.validateHasOneOrMoreTransverseBar(line)).isEqualTo(result);
-    }
-
-    private static Stream<Arguments> validateHasOneOrMoreTransverseBarMethodSource() {
-        return Stream.of(
-                Arguments.of(5, new AllDrawTransverseBarStrategy(), true),
-                Arguments.of(5, new NotAllDrawTransverseBarStrategy(), false),
-                Arguments.of(5, new SwitchDrawTransverseBarStrategy(), true)
-        );
-    }
 
-    @DisplayName("라인 밸리데이션 - 횡단선이 서로 겹치지 않는다.")
+    @DisplayName("라인 생성")
     @ParameterizedTest
-    @MethodSource("validateNotOverlapMethodSource")
-    public void validateNotOverlap(int coutOfPerson, TransverseBarStrategy transverseBarStrategy, boolean result) {
-        List<Boolean> line = Line.drawTransverseBar(coutOfPerson, transverseBarStrategy);
-        assertThat(Line.validateNotOverlap(line)).isEqualTo(result);
+    @MethodSource("createMethod")
+    public void create(int coutOfPerson, TransverseBarStrategy transverseBarStrategy) {
```

Comment:
파라미터 변수 명이 오타난 것 같아요!
```suggestion
    public void create(int countOfPerson, TransverseBarStrategy transverseBarStrategy) {
```